src/Neos.NeosConIo/Configuration/NodeTypes.Talk.yaml

```diff
@@ -3,21 +3,38 @@
     'Neos.Neos:Document': true
     'Neos.NeosConIo:Mixin.EventRelated': true
     'Neos.NodeTypes.BaseMixins:TextMixin': true
+    'Neos.NeosIo:Youtube': true
```

Comment:
I would not inherit here, but directly use 
```
 properties:
    video:
      type: string
      defaultValue: ''
      ui:
        label: 'Youtube video ID'
        reloadIfChanged: true
        inspector:
          group: video

```

IMHO this makes the code more readable :)